### PR TITLE
feat: Upgrade Meteor apps to Shower based deployment

### PR DESCRIPTION
**Pull request by tumido**
Part of: https://github.com/AICoE/meteor/issues/186

### CODE REVIEW

File: argocd/overlays/moc-infra/applications/envs/moc/smaug/operate-first/meteor-shower.yaml [hunk 1]
```diff
@@ -9,8 +9,8 @@ spec:
     namespace: aicoe-meteor
   project: operate-first
   source:
-    path: manifests
-    repoURL: https://github.com/AICoE/meteor
+    path: meteor/overlays/moc/prod
```
Comment (HumairAK) [inline argocd/overlays/moc-infra/applications/envs/moc/smaug/operate-first/meteor-shower.yaml:12]: did you mean `meteor/overlays/moc/smaug` ?

Comment (tumido) [reply argocd/overlays/moc-infra/applications/envs/moc/smaug/operate-first/meteor-shower.yaml]: fixed, thanks! :+1:

**Comment by codificat** [comment]
> /hold for [thoth-station/helm-charts#9](https://github.com/thoth-station/helm-charts/pull/9)

/unhold now that thoth-station/helm-charts#9 is merged

**Pull request closed without merging**